### ISSUE TITLE
01-minimal example does not work as expected.

### ISSUE THREAD

**Issue by zakkg3** [zalando-incubator/kopf#261]
## Expected Behavior
As the https://github.com/zalando-incubator/kopf/blob/master/examples/01-minimal/README.md says:
```
kubectl get kopfexamples
NAME             DURATION   CHILDREN   MESSAGE
kopf-example-1   1m                    hello world
```
and
```
$ kubectl get KopfExample kopf-example-1 -o yaml
apiVersion: zalando.org/v1
kind: KopfExample
metadata:
  ...
spec:
  duration: 1m
  field: value
  items:
  - item1
  - item2
status:
  message: hello world
```

## Actual Behavior
as documented : `Whatever is returned from any handler, is stored in the object’s status under that handler id (which is the function name by default).`
 so I get:
```
#  kubectl get KopfExample kopf-example-1 -o yaml 
...
status:
  create_fn:
    message: hello world
```
and then the message is not picked up:

```
kubectl get kopfexamples
NAME             DURATION   CHILDREN   MESSAGE
kopf-example-1   1m
```

## Steps to Reproduce the Problem

  1. run the minimal example.

## Specifications

  - Platform: metal K8s
  - Kubernetes version: *(use `kubectl version`)* 1.15.5
  - Python version: *(use `python --version`)* 3.8
  - Python packages installed: *(use `pip freeze --all`)* k8s and kopf

```
```
Am I doing something wrong? do I miss something?
thanks!

**Comment by nolar** [comment]
Thanks for reporting. Indeed, that is the definition of `crd.yaml`. It should be:

```yaml
  additionalPrinterColumns:
    - name: Duration
      type: string
      priority: 0
      JSONPath: .spec.duration
      description: For how long the pod should sleep.
    - name: Children
      type: string
      priority: 0
      JSONPath: .status.create_fn.children
      description: The children pods created.
    - name: Message
      type: string
      priority: 0
      JSONPath: .status.create_fn.message
      description: As returned from the handler (sometimes).
```

(With `.create_fn.` in two fields).

**Comment by zakkg3** [comment]
Yes, that will fix all the examples :). Thanks for the super-fast answer. we are thinking about use this framework, and with this response, I think we will go this path. thanks!

**Comment by nolar** [comment]
Released as [`kopf==0.24`](https://github.com/zalando-incubator/kopf/releases/tag/0.24).
